.github/pull_request_template.md

```diff
@@ -0,0 +1,14 @@
+< Jira issue link, if applicable >
+
+### Description
+
+Describe what this PR changes, why we're making the change, how it should be tested...
+
+### Checklist
+
+- [ ] I have signed the [Stream CLA](https://docs.google.com/forms/d/e/1FAIpQLScFKsKkAJI7mhCr7K9rEIOpqIDThrWxuvxnwUq2XkHyG154vQ/viewform) (required)
+- [ ] PR targets the `develop` branch
+- [ ] Changelog updated with client-facing changes
+- [ ] New code is covered by unit tests
+- [ ] Comparison screenshots added for visual changes
+- [ ] Reviewers added
```

Comment:
Not sure, but what about optional checkpoint: "Files written in Java with new changes are converted to Kotlin@

Reply:
I think that's a nice to have, but I wouldn't enact it as a policy to convert to Kotlin if you're touching a file. It's generally harder to review a conversion + changes made at the same time than doing those separately.